randovania/games/dread/json_data/Hanubia.txt

```diff
@@ -874,44 +882,44 @@ EMMI Zone Exit East
 Extra - total_boundings: {'x1': 10400.0, 'x2': 13500.0, 'y1': -2200.0, 'y2': 500.0}
 Extra - polygon: [[13500.0, 500.0], [10400.0, 500.0], [10400.0, -1300.0], [10700.0, -1300.0], [10700.0, -2200.0], [13500.0, -2200.0]]
 Extra - asset_id: collision_camera_013
-> Dock to Total Recharge Station East (dooremmy_002); Heals? False
+> Dock to Total Recharge Station East (Upper); Heals? False
   * Layers: default
-  * EMMI Door to Total Recharge Station East/Dock to EMMI Zone Exit East (dooremmy_002)
+  * EMMI Door to Total Recharge Station East/Dock to EMMI Zone Exit East (Upper)
   * Extra - actor_name: dooremmy_002
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
-  > Event - Break PB Tube (block_pbtube_2)
+  > Event - Break EMMI Zone Exit East PB Tube
       Lay Power Bomb
   > Tunnel to Central Unit
-      After Hanubia - block_pbtube_2
+      After Hanubia - Break EMMI Zone Exit East PB Tube
 
-> Dock to Total Recharge Station East (dooremmy_003); Heals? False
+> Dock to Total Recharge Station East (Lower); Heals? False
   * Layers: default
-  * EMMI Door to Total Recharge Station East/Dock to EMMI Zone Exit East (dooremmy_003)
+  * EMMI Door to Total Recharge Station East/Dock to EMMI Zone Exit East (Lower)
   * Extra - actor_name: dooremmy_003
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
   > Tunnel to Orange EMMI Introduction
       Trivial
 
-> Event - Break PB Tube (block_pbtube_2); Heals? False
+> Event - Break EMMI Zone Exit East PB Tube; Heals? False
   * Layers: default
-  * Event Hanubia - block_pbtube_2
+  * Event Hanubia - Break EMMI Zone Exit East PB Tube
```

Comment:
Just `Hanubia - EMMI Zone Glass Tube`? And if it's not a glass tube, it's even better.